.github/workflows/ci.yml

```diff
@@ -14,12 +14,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['23', '24', '25']
-        rebar3: ['3.16.1']
+        otp: ['25', '26', '27']
```

Comment:
This is a norm for me, but can understand in the context of EMQ if you want to test further back. Let me know.